### PR TITLE
feat: Qdrant support

### PR DESCRIPTION
**Pull request by Anush008**
This PR intends to add [Qdrant](https://qdrant.tech/) as a supported loading destination.

You can find Qdrant setup instructions here.
https://qdrant.tech/documentation/quick-start/

### CODE REVIEW

**Review by mrchtr — COMMENTED** [review]
Thanks @Anush008! Looks good on the first view. Can you add a component test? 
For instance you could have a look at the [chunk_text component](https://github.com/ml6team/fondant/tree/main/components/chunk_text).

Basically, you have to slightly adapt the Dockerfile, and implement your custom test case in the test directory.

File: components/index_qdrant/README.md
Comment (RobbeSneyders) [inline components/index_qdrant/README.md:1]: This file is autogenerated by our precommit hooks. You can add any custom information you want to add (like the "important" note) to the `description` field in the `fondant_component.yaml` file. It supports markdown.

**Review by RobbeSneyders — COMMENTED** [review]
Thanks @Anush008! Great to have this!

This looks good to me apart from running pre-commit. You can run
```bash
pre-commit run --all-files
```
and it will automatically fix the failing pipeline. Note my comment about the README below.

**Review by RobbeSneyders — APPROVED** [review]
Looks good, thanks!

FYI, if there's other Qdrant components you think would make sense, we would be happy to have them. Eg. one to query the vector store.

**Pull request merged**